motionSetPointsApp/src/motionSetPoints.cpp

```diff
@@ -110,7 +110,7 @@ asynStatus motionSetPoints::writeInt32(asynUser *pasynUser, epicsInt32 value)
 // Update the list of available setpoint names
 void motionSetPoints::updatePositions() 
 {
-    const int buffer_size = 4096;
+    const int buffer_size = 8192;
```

Comment:
Also macro here

Reply:
Or rewrite to use an STL string so no limit is required